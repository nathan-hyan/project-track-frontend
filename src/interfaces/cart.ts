export enum CartActions {
  'ADD_TO_CART',
  'REMOVE_FROM_CART',
  'SUBTRACT_ONE_FROM_ITEM',
}
