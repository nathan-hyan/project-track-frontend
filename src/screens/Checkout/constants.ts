export interface Client {
  id: number;
  name: string;
  enterprise: string;
  credit: {
    positiveBalance: number;
    negativeBalance: number;
    mrCredit: number;
  };
}

export const INPUTS = {
  subtotal: 'Subtotal',
  positiveBalance: 'Saldo a favor',
  negativeBalance: 'Saldo deudor',
  delivery: 'Envío',
  discount: 'Descuento',
  totalAmount: 'Monto a pagar',
};

export const PAYMENT_TYPES = [
  {
    id: 0,
    type: 'cash',
    label: 'Efectivo',
  },
  {
    id: 1,
    type: 'debit',
    label: 'Débito',
  },
  {
    id: 2,
    type: 'credit',
    label: 'Crédito',
  },
  {
    id: 3,
    type: 'transfer',
    label: 'Transferencia',
  },
  {
    id: 4,
    type: 'mercadopago',
    label: 'Mercado Pago',
  },
  {
    id: 5,
    type: 'mrCredit',
    label: 'Mundo Regalo Crédito',
  },
];

export const DEFAULT_CLIENT_INFO: Client[] = [
  {
    id: 0,
    name: 'Juan Carlos Gonzalez',
    enterprise: 'Mundo Regalo',
    credit: {
      positiveBalance: 0,
      negativeBalance: 0,
      mrCredit: 0,
    },
  },
];
