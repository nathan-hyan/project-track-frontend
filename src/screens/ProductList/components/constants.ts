import { InputType } from 'components/FormInput';

export type ProductInputName =
  | '_id'
  | 'name'
  | 'internalId'
  | 'price'
  | 'cost'
  | 'stock'
  | 'category'
  | 'image'
  | 'barcode'
  | 'storeBranch'
  | 'storeId';

interface ProductForm {
  id: number;
  label: string;
  name: ProductInputName;
  type: InputType;
}

export const PRODUCT_FORM: ProductForm[] = [
  {
    id: 0,
    label: 'Nombre del producto',
    name: 'name',
    type: InputType.Text,
  },
  {
    id: 1,
    label: 'Precio',
    name: 'price',
    type: InputType.Number,
  },
  {
    id: 2,
    label: 'Costo',
    name: 'cost',
    type: InputType.Number,
  },
  {
    id: 3,
    label: 'Stock',
    name: 'stock',
    type: InputType.Number,
  },
  {
    id: 4,
    label: 'CÃ³digo de barras',
    name: 'barcode',
    type: InputType.Text,
  },
];
