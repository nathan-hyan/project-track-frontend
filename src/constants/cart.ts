import { Product } from 'interfaces/product';

export const DEFAULT_STORE_BRANCH = '6053adb8a6b58a6e35dd4655';
export interface CartProduct {
  item: Product;
  quantity: number;
}

export enum PaymentType {
  List = 'list',
  Cash = 'cash',
}

export interface Payment {
  subtotal: number;
  positiveBalance: number;
  negativeBalance: number;
  delivery: number;
  discount: number;
}

export interface CartInitialStateType {
  products: CartProduct[];
  paymentType: PaymentType;
}

export const cartInitialState: CartInitialStateType = {
  products: [],
  paymentType: PaymentType.List,
};
