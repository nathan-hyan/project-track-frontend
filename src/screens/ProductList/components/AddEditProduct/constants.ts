import { InputType } from 'components/FormInput';

export type ProductInputName =
  | '_id'
  | 'name'
  | 'internalId'
  | 'stock'
  | 'category'
  | 'image'
  | 'barcode'
  | 'storeBranch'
  | 'storeId';

export type PriceInputName = 'cost' | 'list' | 'onlineStore' | 'cash';

export type LocalFormInputName =
  | 'brand'
  | 'businessOwner'
  | 'weight'
  | 'storeBranch'
  | 'storePosition';

export type DimensionsInputName = 'width' | 'height' | 'depth';

export type DescriptionInputName = 'title' | 'description';

export type VariantsInputName = 'color' | 'barcode' | 'stock';

interface Form<T> {
  id: number;
  label: string;
  name: T;
  type: InputType;
}

export const PRODUCT_FORM: Form<ProductInputName>[] = [
  {
    id: 0,
    label: 'Nombre del producto',
    name: 'name',
    type: InputType.Text,
  },
  {
    id: 1,
    label: 'Stock',
    name: 'stock',
    type: InputType.Number,
  },
  {
    id: 2,
    label: 'Código de barras',
    name: 'barcode',
    type: InputType.Text,
  },
];

export const PRICE_FORM: Form<PriceInputName>[] = [
  {
    id: 1,
    label: 'Precio de costo',
    name: 'cost',
    type: InputType.Number,
  },
  {
    id: 2,
    label: 'Precio de lista',
    name: 'list',
    type: InputType.Number,
  },
  {
    id: 3,
    label: 'Precio online',
    name: 'onlineStore',
    type: InputType.Number,
  },
  {
    id: 4,
    label: 'Precio de contado',
    name: 'cash',
    type: InputType.Number,
  },
];

export const LOCAL_INFO_FORM: Form<LocalFormInputName>[] = [
  {
    id: 1,
    label: 'Marca',
    name: 'brand',
    type: InputType.Text,
  },
  {
    id: 2,
    label: 'Dueño de negocio',
    name: 'businessOwner',
    type: InputType.Text,
  },
  {
    id: 4,
    label: 'Sucursal',
    name: 'storeBranch',
    type: InputType.Text,
  },
  {
    id: 5,
    label: 'Ubicación dentro de sucursal',
    name: 'storePosition',
    type: InputType.Text,
  },
  {
    id: 6,
    label: 'Peso (kg)',
    name: 'weight',
    type: InputType.Number,
  },
];

export const DIMENSIONS_FORM: Form<DimensionsInputName>[] = [
  {
    id: 1,
    label: 'Altura',
    name: 'height',
    type: InputType.Number,
  },
  {
    id: 2,
    label: 'Ancho',
    name: 'width',
    type: InputType.Number,
  },
  {
    id: 3,
    label: 'Largo',
    name: 'depth',
    type: InputType.Number,
  },
];

export const SPECIFICATIONS_FORM: Form<DescriptionInputName>[] = [
  {
    id: 1,
    label: 'Titulo',
    name: 'title',
    type: InputType.Text,
  },
  {
    id: 2,
    label: 'Descripcion',
    name: 'description',
    type: InputType.Text,
  },
];

export const VARIANTS_FORM: Form<VariantsInputName>[] = [
  {
    id: 1,
    label: 'Color',
    name: 'color',
    type: InputType.Text,
  },
  {
    id: 2,
    label: 'Barcode',
    name: 'barcode',
    type: InputType.Text,
  },
  {
    id: 3,
    label: 'Stock',
    name: 'stock',
    type: InputType.Number,
  },
];

export const SPECIFICATIONS_FORM_EMPTY = {
  title: '',
  description: '',
};

export const VARIANTS_FORM_EMPTY = {
  color: '',
  barCode: '',
  stock: 0,
};

export const PROVIDER_PRODUCT_CODE_FORM_EMPTY = {
  id: 0,
  name: '',
};

export enum SearchType {
  'BAR_CODE',
  'PRODUCT_ID',
}
