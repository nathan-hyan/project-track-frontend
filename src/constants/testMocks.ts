import { CartProduct } from './cart';

export const testCartProductArrayMultiQuantity: CartProduct[] = [
  {
    item: {
      dimensions: {
        height: 0,
        width: 0,
        depth: 0,
      },
      price: {
        list: 7405.86,
        onlineStore: 6230.5,
        cash: 7030,
        cost: 4019.68,
        lastModified: new Date('2023-01-13T21:51:35.803Z'),
      },
      stock: 0,
      weight: 0,
      storePosition: 'Repisa',
      description: ' Utiliza Broches MIT 50 o Broches MIT 10  Abrocha de 1 a 20 hojas (Br. 50)  Su profundidad máxima de abrochado es de 45mm  Puede utilizarse como clavadora (abierta)  Estructura metálica (máxima durabilidad)  Recubierta con pintura epoxi, niquel y zinc  Diseñada para uso intenso en comercios, oficinas y el hogar',
      internalId: 0,
      _id: '62ab4e5a5976be00160b776b',
      name: 'Abrochadora MIT FX50 Pinza',
      category: 'libreria',
      image: '',
      barcode: '7793928100244',
      storeBranch: 'Av Belgrano 2848',
      storeId: '6053adb8a6b58a6e35dd4655',
      brand: 'MIT',
      businessOwner: 'Mundo Regalo',
      providerProductCode: [
        {
          id: 15804,
          name: 'Libreria San Pablo',
        },
      ],
      specifications: [],
      subCategory: 'Abrochadoras / Broches',
      variants: [],
    },
    quantity: 3,
  },
  {
    item: {
      dimensions: {
        height: 0,
        width: 0,
        depth: 0,
      },
      price: {
        list: 5911.41,
        onlineStore: 4979.716,
        cash: 5620,
        cost: 3212.72,
        lastModified: new Date('2023-01-13T21:16:58.533Z'),
      },
      stock: 0,
      weight: 0,
      storePosition: 'Repisa',
      description: 'Abrochadora',
      internalId: 150,
      _id: '62ab51675976be00160b776e',
      name: 'Abrochadora Grap 24/6 Pinza',
      category: 'libreria',
      image: '',
      barcode: '7798006050305',
      storeBranch: 'Av Belgrano 2848',
      storeId: '6053adb8a6b58a6e35dd4655',
      brand: 'Grap',
      businessOwner: 'Mundo Regalo',
      providerProductCode: [],
      specifications: [],
      variants: [],
    },
    quantity: 2,
  },
  {
    item: {
      dimensions: {
        height: 0,
        width: 0,
        depth: 0,
      },
      price: {
        list: 5529.2,
        onlineStore: 4657.75,
        cash: 5260,
        cost: 3005,
        lastModified: new Date('2023-01-13T21:18:01.688Z'),
      },
      stock: 0,
      weight: 0,
      storePosition: 'Repisa',
      description: 'Abrochadora',
      internalId: 148,
      _id: '62ab51245976be00160b776c',
      name: 'Abrochadora Grap 21/6 Pinza',
      category: 'libreria',
      image: '',
      barcode: '7798006055096',
      storeBranch: 'Av Belgrano 2848',
      storeId: '6053adb8a6b58a6e35dd4655',
      brand: 'Grap',
      businessOwner: 'Mundo Regalo',
      providerProductCode: [],
      specifications: [],
      variants: [],
    },
    quantity: 1,
  },
  {
    item: {
      dimensions: {
        height: 0,
        width: 0,
        depth: 0,
      },
      price: {
        list: 4650.39,
        onlineStore: 3917.45,
        cash: 4420,
        cost: 2527.39,
        lastModified: new Date('2023-01-13T21:18:55.638Z'),
      },
      stock: 0,
      weight: 0,
      storePosition: '0',
      description: 'Nuestra pinza tiene un diseño sólido y dinámico, está compuesta de piezas estampadas en chapa de acero de primera calidad, las cuales no presentan bordes con filos ni ángulos que puedan provocar molestias al usuario. El Cargador, pieza donde se colocan los broches posee un frente cerrado que permite un abrochado perfecto aún en máximas exigencias. Además posee una base templada que asegura su buen funcionamiento. A lo largo de más de 30 años de trayectoria y a través de un estudio desarrollado de su rendimiento podemos afirmar que la Pinza es Confiable. Éste producto tiene un mecanismo simple, con escasas piezas móviles y su forma anátomica permite un uso suave y efectivo. Por su escaso peso que varía entre 200 y 250 gramos según la medida de la pinza y su armonica estructura evita la fatiga en el empleo intensivo. La alimentación es de carga simple, ya que no es necesario abrir la pinza y brinda tiempo al usuario en su acelerado ritmo de trabajo. La abrochadora se presenta pintada con pintura epoxi (Sistema de protección de larga duración) en varios colores, que asegura durabilidad y garantia de calidad. Bajo éstas condiciones logramos un producto competitivo tanto en el orden nacional como el internacional, presentando significativas ventajas respecto a otras máquinas similares.',
      internalId: 0,
      _id: '62ab51435976be00160b776d',
      name: 'Abrochadora Grap 10/50 Pinza',
      category: 'libreria',
      image: '',
      barcode: '7798006055058',
      storeBranch: 'Av Belgrano 2848',
      storeId: '6053adb8a6b58a6e35dd4655',
      brand: 'Grap',
      businessOwner: 'Mundo Regalo',
      providerProductCode: [],
      specifications: [
        {
          title: 'Broches Utilizados',
          description: '10/50',
        },
      ],
      variants: [],
      subCategory: 'Abrochadoras',
    },
    quantity: 5,
  },
];
export const testCartProductArraySingleQuantity: CartProduct[] = [
  {
    item: {
      dimensions: {
        height: 0,
        width: 0,
        depth: 0,
      },
      price: {
        list: 7405.86,
        onlineStore: 6230.5,
        cash: 7030,
        cost: 4019.68,
        lastModified: new Date('2023-01-13T21:51:35.803Z'),
      },
      stock: 0,
      weight: 0,
      storePosition: 'Repisa',
      description: ' Utiliza Broches MIT 50 o Broches MIT 10  Abrocha de 1 a 20 hojas (Br. 50)  Su profundidad máxima de abrochado es de 45mm  Puede utilizarse como clavadora (abierta)  Estructura metálica (máxima durabilidad)  Recubierta con pintura epoxi, niquel y zinc  Diseñada para uso intenso en comercios, oficinas y el hogar',
      internalId: 0,
      _id: '62ab4e5a5976be00160b776b',
      name: 'Abrochadora MIT FX50 Pinza',
      category: 'libreria',
      image: '',
      barcode: '7793928100244',
      storeBranch: 'Av Belgrano 2848',
      storeId: '6053adb8a6b58a6e35dd4655',
      brand: 'MIT',
      businessOwner: 'Mundo Regalo',
      providerProductCode: [
        {
          id: 15804,
          name: 'Libreria San Pablo',
        },
      ],
      specifications: [],
      subCategory: 'Abrochadoras / Broches',
      variants: [],
    },
    quantity: 1,
  },
  {
    item: {
      dimensions: {
        height: 0,
        width: 0,
        depth: 0,
      },
      price: {
        list: 5911.41,
        onlineStore: 4979.716,
        cash: 5620,
        cost: 3212.72,
        lastModified: new Date('2023-01-13T21:16:58.533Z'),
      },
      stock: 0,
      weight: 0,
      storePosition: 'Repisa',
      description: 'Abrochadora',
      internalId: 150,
      _id: '62ab51675976be00160b776e',
      name: 'Abrochadora Grap 24/6 Pinza',
      category: 'libreria',
      image: '',
      barcode: '7798006050305',
      storeBranch: 'Av Belgrano 2848',
      storeId: '6053adb8a6b58a6e35dd4655',
      brand: 'Grap',
      businessOwner: 'Mundo Regalo',
      providerProductCode: [],
      specifications: [],
      variants: [],
    },
    quantity: 1,
  },
  {
    item: {
      dimensions: {
        height: 0,
        width: 0,
        depth: 0,
      },
      price: {
        list: 5529.2,
        onlineStore: 4657.75,
        cash: 5260,
        cost: 3005,
        lastModified: new Date('2023-01-13T21:18:01.688Z'),
      },
      stock: 0,
      weight: 0,
      storePosition: 'Repisa',
      description: 'Abrochadora',
      internalId: 148,
      _id: '62ab51245976be00160b776c',
      name: 'Abrochadora Grap 21/6 Pinza',
      category: 'libreria',
      image: '',
      barcode: '7798006055096',
      storeBranch: 'Av Belgrano 2848',
      storeId: '6053adb8a6b58a6e35dd4655',
      brand: 'Grap',
      businessOwner: 'Mundo Regalo',
      providerProductCode: [],
      specifications: [],
      variants: [],
    },
    quantity: 1,
  },
  {
    item: {
      dimensions: {
        height: 0,
        width: 0,
        depth: 0,
      },
      price: {
        list: 4650.39,
        onlineStore: 3917.45,
        cash: 4420,
        cost: 2527.39,
        lastModified: new Date('2023-01-13T21:18:55.638Z'),
      },
      stock: 0,
      weight: 0,
      storePosition: '0',
      description: 'Nuestra pinza tiene un diseño sólido y dinámico, está compuesta de piezas estampadas en chapa de acero de primera calidad, las cuales no presentan bordes con filos ni ángulos que puedan provocar molestias al usuario. El Cargador, pieza donde se colocan los broches posee un frente cerrado que permite un abrochado perfecto aún en máximas exigencias. Además posee una base templada que asegura su buen funcionamiento. A lo largo de más de 30 años de trayectoria y a través de un estudio desarrollado de su rendimiento podemos afirmar que la Pinza es Confiable. Éste producto tiene un mecanismo simple, con escasas piezas móviles y su forma anátomica permite un uso suave y efectivo. Por su escaso peso que varía entre 200 y 250 gramos según la medida de la pinza y su armonica estructura evita la fatiga en el empleo intensivo. La alimentación es de carga simple, ya que no es necesario abrir la pinza y brinda tiempo al usuario en su acelerado ritmo de trabajo. La abrochadora se presenta pintada con pintura epoxi (Sistema de protección de larga duración) en varios colores, que asegura durabilidad y garantia de calidad. Bajo éstas condiciones logramos un producto competitivo tanto en el orden nacional como el internacional, presentando significativas ventajas respecto a otras máquinas similares.',
      internalId: 0,
      _id: '62ab51435976be00160b776d',
      name: 'Abrochadora Grap 10/50 Pinza',
      category: 'libreria',
      image: '',
      barcode: '7798006055058',
      storeBranch: 'Av Belgrano 2848',
      storeId: '6053adb8a6b58a6e35dd4655',
      brand: 'Grap',
      businessOwner: 'Mundo Regalo',
      providerProductCode: [],
      specifications: [
        {
          title: 'Broches Utilizados',
          description: '10/50',
        },
      ],
      variants: [],
      subCategory: 'Abrochadoras',
    },
    quantity: 1,
  },
];
