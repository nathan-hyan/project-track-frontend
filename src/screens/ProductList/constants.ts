import { toast } from 'react-toastify';
import { MESSAGES } from 'constants/notify';

export const notifications = {
  productDeleted: () => toast.success(MESSAGES.success.productDeleted),
  productNotDeleted: () => toast.error(MESSAGES.error.productNotDeleted),
};
