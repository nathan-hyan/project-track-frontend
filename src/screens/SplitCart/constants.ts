import { PaymentType } from 'constants/cart';

export const OPTIONS = [
  {
    id: 0,
    value: PaymentType.List,
    label: 'Lista',
  },
  {
    id: 1,
    value: PaymentType.Cash,
    label: 'Contado / DÃ©bito',
  },
];
