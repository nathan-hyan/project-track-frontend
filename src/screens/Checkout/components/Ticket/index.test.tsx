import { render } from '@testing-library/react';
import { PaymentType } from 'constants/cart';
import { testCartProductArraySingleQuantity } from 'constants/testMocks';

import { ticketToPrint } from '.';

describe('<Ticket />', () => {
  it('renders correctly', () => {
    expect(
      render(ticketToPrint(testCartProductArraySingleQuantity, PaymentType.Cash)).asFragment(),
    ).toMatchSnapshot();
  });
});
