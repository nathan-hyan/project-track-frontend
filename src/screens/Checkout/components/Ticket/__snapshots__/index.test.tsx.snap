// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Ticket /> renders correctly 1`] = `
<DocumentFragment>
  <div
    class="rtp-printer"
    data-printer-type="epson"
  >
    <div
      class="rtp-text"
      data-align="center"
      data-size-height="2"
      data-size-width="2"
    >
      * Tienda Mundo Regalo *
    </div>
    <br
      class="rtp-br"
    />
    <br
      class="rtp-br"
    />
    <div
      class="rtp-text"
      data-align="center"
    >
      Dirección: Av. Manuel Belgrano 2846, San Miguel de Tucumán
    </div>
    <div
      class="rtp-text"
      data-align="center"
    >
      Cel: (381)316-9319
    </div>
    <hr
      class="rtp-line"
      data-character="="
    />
    <div
      class="rtp-row"
    >
      <div
        class="rtp-text rtp-row-left"
      >
        Cliente:
      </div>
      <div
        class="rtp-text rtp-row-right"
      >
        No especificado
      </div>
    </div>
    <div
      class="rtp-row"
    >
      <div
        class="rtp-text rtp-row-left"
      >
        Nº de Cliente:
      </div>
      <div
        class="rtp-text rtp-row-right"
      >
        0000000000
      </div>
    </div>
    <div
      class="rtp-row"
    >
      <div
        class="rtp-text rtp-row-left"
      >
        Pedido Nº:
      </div>
      <div
        class="rtp-text rtp-row-right"
      >
        0000000000
      </div>
    </div>
    <hr
      class="rtp-line"
    />
    <div
      class="rtp-text"
      data-align="center"
      data-bold="true"
    >
      ** Ticket no válido como factura **
    </div>
    <hr
      class="rtp-line"
    />
    <div
      class="rtp-row"
      data-gap="2"
    >
      <div
        class="rtp-text rtp-row-left"
      >
        Abrochadora MIT FX50 Pinza
      </div>
      <div
        class="rtp-text rtp-row-right"
      >
        $7030.00
      </div>
    </div>
    <div
      class="rtp-row"
      data-gap="2"
    >
      <div
        class="rtp-text rtp-row-left"
      >
        Cantidad: 1
      </div>
      <div
        class="rtp-text rtp-row-right"
      >
        Precio unit.: $7030.00
      </div>
    </div>
    <br
      class="rtp-br"
    />
    <div
      class="rtp-row"
      data-gap="2"
    >
      <div
        class="rtp-text rtp-row-left"
      >
        Abrochadora Grap 24/6 Pinza
      </div>
      <div
        class="rtp-text rtp-row-right"
      >
        $5620.00
      </div>
    </div>
    <div
      class="rtp-row"
      data-gap="2"
    >
      <div
        class="rtp-text rtp-row-left"
      >
        Cantidad: 1
      </div>
      <div
        class="rtp-text rtp-row-right"
      >
        Precio unit.: $5620.00
      </div>
    </div>
    <br
      class="rtp-br"
    />
    <div
      class="rtp-row"
      data-gap="2"
    >
      <div
        class="rtp-text rtp-row-left"
      >
        Abrochadora Grap 21/6 Pinza
      </div>
      <div
        class="rtp-text rtp-row-right"
      >
        $5260.00
      </div>
    </div>
    <div
      class="rtp-row"
      data-gap="2"
    >
      <div
        class="rtp-text rtp-row-left"
      >
        Cantidad: 1
      </div>
      <div
        class="rtp-text rtp-row-right"
      >
        Precio unit.: $5260.00
      </div>
    </div>
    <br
      class="rtp-br"
    />
    <div
      class="rtp-row"
      data-gap="2"
    >
      <div
        class="rtp-text rtp-row-left"
      >
        Abrochadora Grap 10/50 Pinza
      </div>
      <div
        class="rtp-text rtp-row-right"
      >
        $4420.00
      </div>
    </div>
    <div
      class="rtp-row"
      data-gap="2"
    >
      <div
        class="rtp-text rtp-row-left"
      >
        Cantidad: 1
      </div>
      <div
        class="rtp-text rtp-row-right"
      >
        Precio unit.: $4420.00
      </div>
    </div>
    <br
      class="rtp-br"
    />
    <hr
      class="rtp-line"
      data-character="="
    />
    <div
      class="rtp-row"
    >
      <div
        class="rtp-text rtp-row-left"
        data-bold="true"
      >
        Total:
      </div>
      <div
        class="rtp-text rtp-row-right"
        data-bold="true"
      >
        $22330.00
      </div>
    </div>
    <hr
      class="rtp-line"
      data-character="="
    />
    <div
      class="rtp-row"
    >
      <div
        class="rtp-text rtp-row-left"
      >
        Medio de pago:
      </div>
      <div
        class="rtp-text rtp-row-right"
      >
        Efectivo
      </div>
    </div>
    <br
      class="rtp-br"
    />
    <div
      class="rtp-text"
      data-align="center"
      data-bold="true"
    >
      ** Gracias por su compra **
    </div>
    <br
      class="rtp-br"
    />
  </div>
</DocumentFragment>
`;
